### PR TITLE
feat: add whiskers & update colors with accents

### PR DESCRIPTION
**Pull request by zspher**
add whiskers and updated colors
fixes: https://github.com/catppuccin/qt5ct/issues/4 & https://github.com/catppuccin/qt5ct/issues/1

before:
Screenshot_2024-10-10-12-47-34
after:
Screenshot_2024-10-10-16-19-38

> to reviewers:
> i've added the `qt5ct` folder in themes in case qt6ct themes will be added here as well
> if that should be a seperate repo that i will remove that folder
>> side note for clarity:
>> qt5ct and qt6ct theme is the same except for
>> - qt5ct - 21x3 color fields 
>> - qt6ct - 22x3 color fields

### CODE REVIEW

File: qt5ct.tera [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+whiskers:
+  version: "2.5.0"
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/qt5ct/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```
Comment (uncenter) [inline qt5ct.tera:7]:
```suggestion
  filename: "themes/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```

Comment (zspher) [reply qt5ct.tera:7]: so qt6ct themes should be a separate repo?

Comment (zspher) [reply qt5ct.tera:7]: qt5ct and qt6ct theme is the same except for

- qt5ct - 21x3 color fields
- qt6ct - 22x3 color fields

Comment (uncenter) [reply qt5ct.tera:7]: Is that like the next version? I imagine we would leave behind a branch with just the v5 themes and then update main to v6. Though, this repository is named specifically with for v5 so...

Comment (uncenter) [reply qt5ct.tera:7]: That would be a more meta issue for @sgoudham. I'm not sure if we would rename the repository to something more general like `qtct`, or just note that we support both in the readme, or make a new repository entirely...

Comment (zspher) [reply qt5ct.tera:7]: yup, ok 
for now i'll just remove the folder

**Comment by zspher** [comment]
@uncenter is there anything blocking this PR?

**Comment by sgoudham** [comment]
Hey there :wave:,

I recently tested out the theme from the files in this PR and, while I believe it's an improvement, it doesn't feel "Catppuccin" to me.

E.g.
image

For the image above, I'd expect:

- The "Preview Window" text should be `base` since it's on an accented background.
- The background should be `base` instead of `mantle`
- The pane which has the "Tabs" should be `base`
- We don't have a defined style guide for search bars but I believe catppuccin/userstyles uses `surface0` which should look good.

Are we able to make those changes so that the preview falls more in line with other applications in the organisation?

**Comment by zspher** [comment]
so i've done some testing

1. i can't really change "Preview Widow" by changing any foreground/text color
2. you are using the fusion qt style which is the only default style that does not use the "normal background" color, which is set to base. Instead fusion uses the "button background" color.
3. I can't really change the search bar. That uses the "normal background" color in all tested qt styles

*Breeze*
Screenshot_2024-10-19-23-14-41
*lightly* (not a default installed qt style, thought recommended in readme)
Screenshot_2024-10-19-23-14-54
*windows*
Screenshot_2024-10-19-23-15-05
*fusion*
Screenshot_2024-10-19-23-15-13

**Comment by zspher** [comment]
@sgoudham any updates on this

**Comment by shadow-absorber** [comment]
this looks like a good pull request to shadow and don't see any major problems

File: qt5ct.tera (hunk 1 above)
Comment (shadow-absorber) [reply qt5ct.tera]: just opened this on the main catppuccin github discussion page: [https://github.com/catppuccin/catppuccin/discussions/2624](https://github.com/catppuccin/catppuccin/discussions/2624)

**Comment by sgoudham** [comment]
> @sgoudham any updates on this

I've been busy and forgot about this, sorry!
I'm happy to merge given that it seems we can't do much.

File: qt5ct.tera [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+whiskers:
+  version: "2.5.0"
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```
Comment (sgoudham) [inline qt5ct.tera:7]: Do the filenames need to be capitalised? I'd like to standardise on lowercase in the organisation (i.e. `catppuccin-mocha-mauve.conf`)

```suggestion
  filename: "themes/catppuccin-{{ flavor.identifier }}-{{ accent }}.conf"
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
One comment from me, could you also update the README to mention the accents and remove the section in the FAQ since accents are supported by default with this PR.

**Comment by zspher** [comment]
> One comment from me, could you also update the README to mention the accents and remove the section in the FAQ since accents are supported by default with this PR.

done

**Review by sgoudham — APPROVED** [review]
Sorry for the wait on this, thanks for all your work!

**Pull request merged**